db/buckets.go

```diff
@@ -33,6 +33,7 @@ const (
 	BlockCommitments
 	Temporary // used temporarily for migrations
 	SchemaIntermediateState
+	L1HandlerTxnHash // maps l1 handler msg hash to l1 handler txn hash
```

Comment:
please run `make generate` it should update `buckets_enumer.go` file automatically + seems like we need migration

Reply:
I've ran `make generate` to update the `buckets_enumer.go`. Just need to think about and handle the migration.

Reply:
added a migration function to populate the new bucket for historical blocks that exist in the db